docs/buildah-build.1.md

```diff
@@ -288,6 +288,61 @@ The [username[:password]] to use to authenticate with the registry if required.
 If one or both values are not supplied, a command line prompt will appear and the
 value can be entered.  The password is entered without echo.
 
+**--cw** *options*
+
+Produce an image suitable for use as a confidential workload running in a
+trusted execution environment (TEE) using krun.  Instead of the conventional
```

Comment:
can we add a man page reference to krun at the bottom of this page?

Reply:
The package that provides `/usr/bin/krun` doesn't provide a man page for it for us to reference.  Is there an alternate one you'd suggest?